### DISCUSSION TITLE
Using Pro Backpack in local computer

### DISCUSSION THREAD

**Discussion by lanngk2** [Laravel-Backpack/community-forum#1089] [Q&A (Help)]
Hi everyone,

I’ve just purchased the Pro Backpack and want to test it on my local computer before going live with a domain. I’m using XAMPP. How can I set this up? Thanks!

**Comment by pxpm** [comment]
Hey @panadollr 

If you go to your backpack account at https://backpackforlaravel.com/user/tokens in the tab: `Instructions` you will find all the information you need to setup any package. 

Basically after configuring the auth to our private servers, a `composer require backpack/pro` should get you the package. 

Let me know if that helps.

Cheers

**Reply by lanngk2** [reply]
I’m concerned that using the license code on a local machine for testing might prevent me from using it on the domain later.

**Reply by jcastroa87** [reply]
Hello you need to use your credentials, as @pxpm says, follow the "Instructions" in "Tokens" section as you will use not in production mode, will be ok to use in your domain later.

Cheers.
